### PR TITLE
Updated Microsoft-Office-365-Email-Indicators-to-Slack

### PR DESCRIPTION
**Pull request by mcochran-r7**
Updated plugin versions replaced Python with Type Converter, added parameters

## Requirements

Workflow contributors, verify you have completed the following items by checking them off:

### Checklist
- [ ] Workflow verified as working in a test
- [ ] Workflow bundle includes `Name_of_Workflow.icon` file, `help.md` file, `workflow.spec.yaml` file, and `screenshots` directory
- [ ] Workflow title should follow general [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [ ] Workflow includes a detailed Description
- [ ] Workflow step names follow [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [ ] Triggers, Decisions, and other workflow steps include descriptions where needed
- [ ] Workflow includes at least one `Final Report` artifact that summarizes workflow execution details
- [ ] Workflow bundle (directory) is named after the title of the workflow in snake case with first letters capitalized, e.g. `workflows/Name_of_Workflow/`
- [ ] `help.md` file includes Description, Key Features, Requirements, Setup, Technical Details, Version History, and References
- [ ] `workflow.spec.yaml` file includes `products`, `name`, `title`, `description`, `use_cases` tags, `keywords` tags, `source_url`, and the `name` for each screenshot matches a PNG image in the `screenshots` directory

### Assessment

Post your results with screenshots in this PR using the following style:

```
### Screenshots

Screenshots help your reviewers provide valuable feedback on your contribution. To add a screenshot here, simply drag and drop the PNG files from a file explorer.

#### Workflow Builder

<Insert Workflow Image Here>

#### Job

<Insert Job Image Here>

#### Artifact

<Insert Artifact Image Here>
```

### CODE REVIEW

**Comment by mcochran-r7** [comment]
Working version in SimLab (https://us.connect.insight.rapid7.com/7F913342E90748B5F07A#/workflows/details/4ea5604b-9790-479c-b436-e79c5d1b5a68/versions/dbca436e-f175-4e31-b559-4651c51ffb1d)

File: workflows/Microsoft_Office_365_Email_Indicators_to_Slack/Microsoft_Office_365_Email_Indicators_to_Slack.icon [hunk 1]
```diff
@@ -707,11 +653,11 @@
                 "flatten_attachments": false,
                 "folder_name": "Inbox",
                 "interval": 15,
-                "mailbox_id": "jschipp@komanddev.onmicrosoft.com",
-                "subject_query": ""
+                "mailbox_id": "{{$workflow.[Email_Address]}}",
+                "subject_query": "Update Test Slack"
```
Comment (ewilson-r7) [inline workflows/Microsoft_Office_365_Email_Indicators_to_Slack/Microsoft_Office_365_Email_Indicators_to_Slack.icon:657]: Probably want to remove the Subject Query for the mailbox trigger here

**Review by ewilson-r7 — CHANGES_REQUESTED** [review]

File: workflows/Microsoft_Office_365_Email_Indicators_to_Slack/Microsoft_Office_365_Email_Indicators_to_Slack.icon (hunk 1 above)
Comment (mcochran-r7) [reply workflows/Microsoft_Office_365_Email_Indicators_to_Slack/Microsoft_Office_365_Email_Indicators_to_Slack.icon]: Removed and updated

**Review by ewilson-r7 — APPROVED** [review]
Good to go

**Pull request merged**